transaction_response.go

```diff
@@ -91,6 +91,10 @@ func (response TransactionResponse) GetRecord(client *Client) (TransactionRecord
 		SetNodeAccountIDs([]AccountID{response.NodeID}).
 		Execute(client)
 
+	for receipt.Status == StatusThrottledAtConsensus {
+		receipt, err = retryTransaction(client, response.Transaction)
+	}
+
```

Comment:
Maybe this could be verified with:
Creating a test which constantly tries to get a record of a transaction. But could be a loss of CI time as we have to bake a certain amount of time which we would constantly loop over a transaction being sent and getting its record? WDYT?

Reply:
Yeah, this would slow down a lot CI time, since the throttles are quite random.